bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/nifs/NifsRule.java

```diff
@@ -191,29 +193,31 @@ public IP2Problems check(ObservationElements elements)  {
                 if (baseTarget == null || oiTarget.isEmpty() || aoTarget.isEmpty()) return null;
                 //now, let's compare coordinates. If they are the same, raise an error
 
-                double baseC1 = baseTarget.getTarget().getRaHours();
-                double baseC2 = baseTarget.getTarget().getDecDegrees();
+                final Option<Long> when = elements.getSchedulingBlock().map(SchedulingBlock::start);
```

Comment:
Yes, Gemini `Option` right now. I guess we should replace it with Java `Optional` at some point.